frontend/app/component/form/data_entry/voters_and_votes/VotersAndVotesForm.test.tsx

```diff
@@ -73,25 +73,26 @@ function renderForm(defaultValues: Partial<PollingStationValues> = {}) {
   );
 }
 
-function getFormFields(includingRecountedFields: boolean = false): HTMLElement[] {
-  const fields: HTMLElement[] = [
-    screen.getByTestId("poll_card_count"),
-    screen.getByTestId("proxy_certificate_count"),
-    screen.getByTestId("voter_card_count"),
-    screen.getByTestId("total_admitted_voters_count"),
-    screen.getByTestId("votes_candidates_count"),
-    screen.getByTestId("blank_votes_count"),
-    screen.getByTestId("invalid_votes_count"),
-    screen.getByTestId("total_votes_cast_count"),
-  ];
-  if (includingRecountedFields) {
-    fields.push(screen.getByTestId("poll_card_recount"));
-    fields.push(screen.getByTestId("proxy_certificate_recount"));
-    fields.push(screen.getByTestId("voter_card_recount"));
-    fields.push(screen.getByTestId("total_admitted_voters_recount"));
-  }
-  return fields;
-}
+const votersFieldIds = {
+  pollCardCount: "poll_card_count",
+  proxyCertificateCount: "proxy_certificate_count",
+  voterCardCount: "voter_card_count",
+  totalAdmittedVotersCount: "total_admitted_voters_count",
+};
+
+const votesFieldIds = {
+  votesCandidatesCount: "votes_candidates_count",
+  blankVotesCount: "blank_votes_count",
+  invalidVotesCount: "invalid_votes_count",
+  totalVotesCastCount: "total_votes_cast_count",
+};
+
+const recountFieldIds = {
+  pollcardRecount: "poll_card_recount",
```

Comment:
```suggestion
  pollCardRecount: "poll_card_recount",
```

Reply:
done in 576e1960c8d5d40c27e47c10b81c06ff7d3bc9df